.jshintrc

```diff
@@ -40,6 +40,7 @@
         "customMatchers": true,
         "KEYS": true,
         "MAX_SAFE_INTEGER": true,
-        "SUPPORTED_INPUT_TYPES": true
+        "SUPPORTED_INPUT_TYPES": true,
+        "window": true
```

Comment:
You should use Angular's `$window` service, instead of the global variable `window`.